app/views/shared/_further_info_trigger.html.erb

```diff
@@ -0,0 +1,5 @@
+<a href="" class="further-info__trigger" data-further-info-trigger>
+  <svg class="svg-icon svg-icon-tooltip" xmlns="http://www.w3.org/2000/svg" style="">
```

Comment:
`href=""` – can this have a fallback? Or is it hidden without js?

Might still be worth doing either way, if for whatever reason there was an error in the js, but the browser was js-enabled, this would be visible and have odd behaviour. 

Perhaps where it's linking to can be given an ID and this link just targets that.